apps/actions-dependencies-updater/src/workflows.mts

```diff
@@ -398,3 +391,85 @@ function extractActionIdentifiersFromSteps(
     .filter((s): s is { uses: string } => !!s.uses)
     .map(({ uses }: { uses: string }) => uses);
 }
+
+/**
+ * An edge-case for parsing jobs which invoke a workflow_call. Which essentially references another workflow file, local or remote.
+ */
+async function parseWorkflowCall(
```

Comment:
Will this or should this catch using other workflows like this?

```
jobs:
  call_remote_workflow:
    uses: org-name/repo-name/.github/workflows/remote-workflow.yml@main
    with:
      example_input: 'Hello from another repo!'
    secrets:
      EXAMPLE_SECRET: ${{ secrets.EXAMPLE_SECRET }}
```

Reply:
Yeah, that's exactly what this method is designed for.

It's getting kinda messy to track actions/workflows local to an external action/workflow, so this method is kind of stubbed. This is something I should deal with if this tool sees more uses, but for now it's unfortunately a little janked given the timeline.